Cookbook/Technical-Documents/ReleaseProcess.md

```diff
@@ -55,7 +55,7 @@ There are usually two release engineers working at any given time. It goes witho
 *It starts after all opened issues had been adressed and can take several cycles until QA's approval*
 
 1. Triger a new release build in the `#ios-build` channel
-1. Obtain the release notes from the Product Manager and update them in the [AppStoreConnect](https://appstoreconnect.apple.com)
+1. Obtain the release notes from the Product Manager and update them in the [AppStoreConnect](https://appstoreconnect.apple.com). Be aware that the Release Notes (and all the App Record) has 2 localized versions: English UK (which containes NHS mentions) and another English (Australian by August/2019) for the other Territories. Paste the Release Notes on both. 
```

Comment:
Some friendly suggestions. Lower case 'release notes' to match earlier use of the phrase.

```suggestion
1. Obtain the release notes from the Product Manager and update them in the [AppStoreConnect](https://appstoreconnect.apple.com). Be aware that the release notes (and all the App Record) has 2 localized versions: English UK (which contains references to the NHS) and another English (Australian by August/2019) for the other territories. Paste the release notes on both. 
```